### PR TITLE
Provide HUB -> RLPADDR with the correct `init_code_hash`

### PR DESCRIPTION
**Pull request by OlivierBBB**

### CODE REVIEW

File: hub/instruction_handling/create/shorthands.tex [hunk 1]
```diff
@@ -2,28 +2,28 @@
...
+		\locMxpx             & \define & \stackMxpx          _{i - \createFirstStackRowOffset}  \\
+		\locOogx             & \define & \stackOogx          _{i - \createFirstStackRowOffset}  \\
+		\locOffsetHi         & \define & \stackItemValHi{1}  _{i - \createFirstStackRowOffset}  \\
+		\locOffsetLo         & \define & \stackItemValLo{1}  _{i - \createFirstStackRowOffset}  \\
+		\locSizeHi           & \define & \stackItemValHi{2}  _{i - \createFirstStackRowOffset}  \\
+		\locSizeLo           & \define & \stackItemValLo{2}  _{i - \createFirstStackRowOffset}  \\
+		\locCnWillRevert     & \define & \cnWillRev          _{i - \createFirstStackRowOffset}  \\
+		\locCnRevertStamp    & \define & \cnRevStamp         _{i - \createFirstStackRowOffset}  \\
+		\locHashInfoKeccakHi & \define & \stackHashInfoValHi _{i - \createFirstStackRowOffset}  \\
+		\locHashInfoKeccakLo & \define & \stackHashInfoValLo _{i - \createFirstStackRowOffset}  \\
```
Comment (OlivierBBB) [inline hub/instruction_handling/create/shorthands.tex:18]: Better name

File: hub/instruction_handling/create/generalities/unexceptional.tex [hunk 2]
```diff
@@ -60,23 +60,62 @@
...
-			\end{array} \right]
+			\left\{ \begin{array}{lcl}
+				\locRlpAddrRecipe & \define &
+				\left[ \begin{array}{lrcl}
+					+ & \locIsCreate         & \!\!\! \cdot \!\!\! & \addressCreationRecipeConstantStd \\
+					+ & \locIsCreateTwo      & \!\!\! \cdot \!\!\! & \addressCreationRecipeConstantTwo \\
+				\end{array} \right] \vspace{2mm} \\
+				\locInitCodeHashHi & \define & \locIsCreateTwo \cdot
+				\left[ \begin{array}{rcl}
+					\locTriggerHashInfo       & \cdot & \locHashInfoKeccakHi \\
```
Comment (lorenzogentile404) [inline hub/instruction_handling/create/generalities/unexceptional.tex:82]: Screenshot 2024-12-03 at 18 24 33
this expression seems to be not well-formed.

Comment (OlivierBBB) [reply hub/instruction_handling/create/generalities/unexceptional.tex:82]: Yes, it's missing `+` signs, will fix.

Comment (OlivierBBB) [reply hub/instruction_handling/create/generalities/unexceptional.tex]: Done

**Pull request merged**